src/main/resources/org/terasology/launcher/views/application.fxml

```diff
@@ -130,9 +130,9 @@
           <center>
             <AnchorPane prefHeight="200.0" prefWidth="200.0">
               <children>
-                <VBox alignment="BOTTOM_CENTER" prefHeight="399.0" prefWidth="969.0" AnchorPane.bottomAnchor="8.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="8.0">
+                <VBox alignment="BOTTOM_CENTER" prefHeight="399.0" prefWidth="969.0" AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0">
```

Comment:
Please remove the changes in this file- they revert [https://github.com/MovingBlocks/TerasologyLauncher/commit/8d54f83d55d0e0b66b0cc88aeae3d7c17d88417d](https://github.com/MovingBlocks/TerasologyLauncher/commit/8d54f83d55d0e0b66b0cc88aeae3d7c17d88417d) and are most likely unintended.